inc/inventoryaction.class.php

```diff
@@ -111,7 +112,7 @@ public static function runPartialInventory(Agent $agent, $fromMA = false)
                 // not authorized
                 return self::handleAgentResponse($response, $endpoint);
             }
-        } catch (\GuzzleHttp\Exception\ClientException $e) {
+        } catch (\GuzzleHttp\Exception\ClientException $e) { // @phpstan-ignore-line
```

Comment:
Perhaps we don't need the catch, because if the try succeeds it already makes a return.
```suggestion
        }
```